Packs/Reco/ReleaseNotes/1_1_6.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Reco
+
+- Change file_id to asset_id in the `reco` integration. This is a breaking change.
```

Comment:
Since there are breaking changes, please write proper "breaking changes" release notes (using the `-bc` flag - `demisto-sdk update-release-notes -i Packs/Reco/ReleaseNotes/1_1_6.md -bc`).

For more information & docs see:
https://xsoar.pan.dev/docs/documentation/release-notes#breaking-changes-version